tests/integration_tests/ingest/test_ingest.py

```diff
@@ -12,7 +12,9 @@ def test_ingest_when_ok(data_db: Session, client: TestClient, user_header_token)
 
     response = client.post(
         "/api/v1/ingest/UNFCCC.corpus.i00000001.n0000",
-        files={"new_data": open("tests/integration_tests/ingest/test.json", "rb")},
+        files={
+            "new_data": open("tests/integration_tests/ingest/test_bulk_data.json", "rb")
```

Comment:
Can we join the test path properly using os.path.join please to make sure the path parts are strung together in a way that is cross platform supported?
`import os`
```suggestion
            "new_data": open(os.path.join("tests", "integration_tests", "ingest", "test_bulk_data.json"), "rb")
```